src/arrayMethodReduce.js

```diff
@@ -5,7 +5,21 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let accumulator = initialValue;
+    let index = 0;
```

Comment:
```suggestion
    let startIndex = 0;
```
Maybe this one will be more obvious.